packaging/python/build_static_readline_osx.bash

```diff
@@ -4,54 +4,50 @@ set -xe
 #
 # PREREQUESITES:
 #  - curl
-#  - wget
 #  - C/C++ compiler
 #  - /opt/nrnwheel folder created with access rights (this specific path is kept for consistency wrt `build_wheels.bash`)
 
-set -e
-
-if [[ `uname -s` != 'Darwin' ]]; then
+if [[ "$(uname -s)" != 'Darwin' ]]; then
     echo "Error: this script is for macOS only. readline is already built statically in the linux Docker images"
     exit 1
 fi
 
-NRNWHEEL_DIR=/opt/nrnwheel
+NRNWHEEL_DIR="${1:-/opt/nrnwheel/$(uname -m)}"
```

Comment:
nitpick: I'd assign`uname -m` to a var to avoid needing to remember this is arch.